ui/src/main/java/org/jtaccuino/core/ui/api/SheetEvent.java

```diff
@@ -35,7 +35,9 @@ public Sheet getSource() {
         return (Sheet) super.getSource();
     }
 
+    @Override
+    @SuppressWarnings("unchecked") // TODO: Check if it is used and remove if not
     public EventType<SheetEvent> getEventType() {
-        return getEventType();
+        return (EventType<SheetEvent>) super.getEventType();
     }
```

Comment:
Remove, comment out, or ignore?
Method appears unused so the more refined return type is unused.